package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@briebug/cypress-schematic",
-  "version": "3.0.1",
+  "version": "3.0.3",
```

Comment:
You don't need to update the version number here, I have it update automatically when I publish